src/SDK/Language/Flutter.php

```diff
@@ -223,6 +223,12 @@ public function getFiles()
                 'template'      => 'flutter/lib/src/service.dart.twig',
                 'minify'        => false,
             ],
+            [
+                'scope'         => 'default',
+                'destination'   => '/lib/src/model.dart',
```

Comment:
let's keep it inside `/lib/src/models`

Reply:
first time dealing with twig so I just tried to emulate the structure of the services. Thank you!